File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/worksheet/ConvertWorksheetToImage.java
package com.aspose.cells.cloud.examples.worksheet;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class ConvertWorksheetToImage {

    public static void main(String... args) throws IOException {
        // ExStart: convert-worksheet-to-image
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            String input = "Sample1.xlsx";
            Path inputFile = Utils.getPath(ConvertWorksheetToImage.class, input);
            String output = "output.png";
            Path outputFile = Utils.getPath(ConvertWorksheetToImage.class, output);

            String sheetName = "Sheet1";
            String format = "png";
            Integer verticalResolution = null;
            Integer horizontalResolution = null;

            com.aspose.cells.model.ResponseMessage apiResponse = cellsApi.GetWorkSheetWithFormat(input, sheetName,
                    format, verticalResolution, horizontalResolution, Utils.STORAGE, Utils.FOLDER);

            Files.copy(apiResponse.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);
        }

        catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: convert-worksheet-to-image
    }
}

File: Examples/DotNet/CSharp/Worksheet/ConvertWorksheetToImage.cs
using System;
using Com.Aspose.Cells.Api;
using Com.Aspose.Cells.Model;
using Com.Aspose.Storage.Api;

namespace Worksheet
{
    class ConvertWorksheetToImage
    {
        public static void Run()
        {
            // ExStart:1
            CellsApi cellsApi = new CellsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "Sample_Test_Book";
            String fileExtension = ".xls";
            String sheetName = "Sheet1";
            String format = "png";
            int? verticalResolution = null;
            int? horizontalResolution = null;
            String storage = "";
            String folder = "";

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName + fileExtension));

                // Invoke Aspose.Cells Cloud SDK API to convert worksheet to image
                ResponseMessage apiResponse = cellsApi.GetWorkSheetWithFormat(fileName + fileExtension, sheetName, format, verticalResolution, horizontalResolution, storage, folder);

                if (apiResponse != null)
                {
                    Console.WriteLine("Worksheet converted to Image!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}
